### ISSUE TITLE
Loading the config cause `Critical dependency: the request of a dependency is an expression`

### ISSUE THREAD

**Issue by mbellerose** [honeybadger-io/honeybadger-js#1351]
## What are the steps to reproduce this issue?
1. Setup a Next.js project (our project is using 14.1.3)
2. Add Honeybadger following https://docs.honeybadger.io/lib/javascript/integration/nextjs/
3. Run next dev
4. Load a page

## What happens?
Each time a new page is loaded the logs below are generated

## What were you expecting to happen?
The dependency is loaded without the logging noise

## Any logs, error output, etc?
```
 ⚠ ./node_modules/@honeybadger-io/js/dist/server/honeybadger.js
Critical dependency: the request of a dependency is an expression

Import trace for requested module:
./node_modules/@honeybadger-io/js/dist/server/honeybadger.js
[omitted]
```

## Any other comments?
Honeybadger still seems to be working, but the logging noise isn't great.

I believe this was introduce with PR #1268 specifically this [line](https://github.com/honeybadger-io/honeybadger-js/pull/1268/files#diff-a238576a4dd76ae3f749332d8c8383b877bd4ea24267197419505936bc080bd0R83).

## What versions are you using?
**Operating System:** Intel MacOS 13.6.6 (22G630)
**Package Name:** @honeybadger-io/nextjs
**Package Version:** 5.8.14
**Browser Version:** n/a

**Comment by mbellerose** [comment]
Thanks for taking at look at this @subzero10. I took a look at the PR and read the Next.js page linked. I tested this in my project by adding the following to my `next.config.js` and the logs did go away.

```
  experimental: {
    serverComponentsExternalPackages: ['@honeybadger-io/js'],
  }
  ```

I think you're PR to automatically add the package will save some configuration. At the same time I'm happy to update my project's configuration so you're PR isn't needed if you think that is better for the Honeybadger project.

Thanks for figuring this out.
